src/contexts/Balances/Provider.tsx

```diff
@@ -80,6 +83,9 @@ const Provider: React.FC = ({ children }) => {
       setDpiBalance(
         new BigNumber(balances[2]).dividedBy(new BigNumber(10).pow(18))
       )
+      setFliBalance(
+        new BigNumber(balances[2]).dividedBy(new BigNumber(10).pow(18))
```

Comment:
need to increment the array index here and index for all balances afterwards (below)

Reply:
good catch!